core/market/decentralized/src/matcher/matcher.rs

```diff
@@ -17,7 +17,7 @@ use crate::db::models::{SubscriptionId, SubscriptionParseError, SubscriptionVali
 use crate::db::*;
 use crate::migrations;
 use crate::protocol::{
-    Discovery, DiscoveryError, DiscoveryInitError, Propagate, StopPropagateReason,
+    Discovery, DiscoveryError, DiscoveryInitError, ShouldPropagate, StopPropagateReason,
```

Comment:
I don't know if it adds any value...
Note: How it was used 
`Propagate::True` or `Propagate::False`
It was obvious what it does, so why change it?
But let's leave it